web/src/DebugDemandMode.svelte

```diff
@@ -71,14 +118,21 @@
     {:else}
       <p>Hover on a zone</p>
     {/if}
+
+    <hr />
+    <p>
```

Comment:
Panning between neighborhoods causes the left column content to be really frenetic. 

I think moving the fixed height content (like the color ramp) above the other text would help,  or moving the color ramp over the map?

Reply:
Good point... I moved the ramp above the other text. Maybe it should still be on the map, if we wind up consistently showing legends on the map.